pkg/generator/parameters.go

```diff
@@ -61,3 +63,22 @@ func (opts *GenerateParametersOptions) genParameterSet(fn generator) (parameters
 
 	return pset, nil
 }
+
+func getDefaultParamValue(bun bundle.Bundle, name string) (string, error) {
+	for p, v := range bun.Parameters {
+		if p == name {
+			def, ok := bun.Definitions[v.Definition]
+			if !ok {
+				return "", fmt.Errorf("unable to find definition for parameter %s", name)
+			}
+			if def == nil {
+				return "", fmt.Errorf("parameter definition for %s is empty", name)
+			}
+
+			if def.Default != nil {
```

Comment:
We can return the default value directly without any comparison or formatting, because we will render the value for display in the `genSurvey` function.